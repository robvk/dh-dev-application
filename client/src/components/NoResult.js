import React from "react";

import "./NoResult.css";

function NoResult(props: {}) {
  return (
    <div className="no-result">
      <span className="no-result-shrug">¯\_(ツ)_/¯</span>
      <span className="no-result-text">No results found</span>
    </div>
  );
}

export default NoResult;
