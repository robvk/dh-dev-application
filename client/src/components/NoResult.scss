@import "../appColors";

.no-result {
  flex-direction: column;
  display: flex;
  text-align: center;
}

.no-result-shrug {
  font-size: 40px;
}

.no-result-text {
  color: $disabled-color;
}
