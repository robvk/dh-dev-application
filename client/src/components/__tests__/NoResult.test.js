import React from "react";
import ReactDOM from "react-dom";

import NoResult from "../NoResult";

it("renders correctly", () => {
  const wrapper = shallow(<NoResult />);
  expect(wrapper).toMatchSnapshot();
});
