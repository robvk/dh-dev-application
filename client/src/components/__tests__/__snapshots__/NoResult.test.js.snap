// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="no-result"
>
  <span
    className="no-result-shrug"
  >
    ¯\\_(ツ)_/¯
  </span>
  <span
    className="no-result-text"
  >
    No results found
  </span>
</div>
`;
